discovery-provider/src/models/users/audio_transactions_history.py

```diff
@@ -0,0 +1,38 @@
+import enum
+
+from sqlalchemy import BigInteger, Column, DateTime, Enum, String, text
+from src.models.base import Base
+from src.models.model_utils import RepresentableMixin
+
+
+class TransactionType(str, enum.Enum):
+    tip = "TIP"
+    reward = "REWARD"
+    transfer = "TRANSFER"
+    purchase_stripe = "PURCHASE_STRIPE"
+    purchase_coinbase = "PURCHASE_COINBASE"
+
+
+class TransactionMethod(str, enum.Enum):
+    send = "SEND"
+    receive = "RECEIVE"
+    user = "USER"
+    trending = "TRENDING"
+
+
+class AudioTransactionsHistory(Base, RepresentableMixin):
+    __tablename__ = "audio_transactions_history"
+
+    user_bank = Column(String, primary_key=True, nullable=False)
```

Comment:
nit: can we define this primary key separately so it's clear the order of the composite key?

Reply:
this might not be possible actually, nvm. but we can make sure the order here matches the order in the db

Reply:
I think they do match currently?

Reply:
`user_bank` is first in the pkey?